### PR TITLE
Incoming SMS handling for netgear_lte

### PR DESCRIPTION
**Pull request by amelchio**
**Description:**

Add an event and a service call.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#22402

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/netgear_lte.markdown [hunk 1]
```diff
@@ -102,3 +102,50 @@ If you do not supply `notify` configuration, a default notification service with
...
+## {% linkable_title Events %}
+
+### {% linkable_title Event `netgear_lte_sms` %}
+
+Messages arriving in the modem inbox are sent as events of type `netgear_lte_sms` with the following content.
+
+| Event data attribute | Description                              |
+| -------------------- | ---------------------------------------- |
+| `host`               | The modem that received the message.
+| `sms_id`             | The inbox IDs of the received message.
```
Comment (syssi) [inline source/_components/netgear_lte.markdown]: The inbox ID of the received message.

File: source/_components/netgear_lte.markdown [hunk 2]
```diff
@@ -102,3 +102,50 @@ If you do not supply `notify` configuration, a default notification service with
...
+## {% linkable_title Services %}
+
+### {% linkable_title Service `netgear_lte.delete_sms` %}
+
+The integration makes a service call available to delete messages from the modem inbox. This can be used to clean up after incoming SMS events.
+
+| Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `host`                 | no       | The modem that should have a message deleted.
+| `sms_id`               | no       | The inbox IDs of messages to delete.
```
Comment (syssi) [inline source/_components/netgear_lte.markdown]: Same here.

File: source/_components/netgear_lte.markdown (hunk 1 above)
Comment (amelchio) [reply source/_components/netgear_lte.markdown]: Thank you, fixed.

**Review by klaasnicolaas — APPROVED** [review]
✅ Can be merged after merging the parent PR 👍

**Pull request merged**